content/_releases/22/20.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Product Update - v22.20
+description: Details of the product update v22.20 from 19th of May 2022.
+layout: rel
+version: "22.20"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-05-19
+redirect_from:
+  - /releases/22.20.html
+---
+
+## Improvements and Updates
+
+### Re-authentication for sensitive actions
+
+While working on the platform user interface we try to click our way through
+setup options and there is a chance we might accidentally remove a sensitive data.
```

Comment:
While working on the platform interface it is possible that users can accidentally change or delete important or sensitive information.

Reply:
Done